bin/sl-deploy.js

```diff
@@ -144,11 +150,20 @@ if (!local) {
 function exit(err, service) {
   var svc = service ? (service.name || service.id) : serviceName;
   if (err) {
-    console.error('Failed to deploy `%s` as `%s` to `%s` via `%s`',
-                  branchOrPack, svc, baseURL, exit.url);
```

Comment:
strong-globalize supports both glb.error('this {1} is a {2}, ['statue', 'dog']) and glb.error('this {statue} is a {dog}, {statue: 'statue', dog: 'dog'}).  i don't think we want to advertise the former notation, do we?